index.html

```diff
@@ -0,0 +1,105 @@
+      <div class="content-nav">
+        <p class="hint">"If everyone is thinking alike, then no one is thinking."</p>
+        <a href="#" class="logo">
+          <h1>The Power within us 2022</h1>
+        </a>
+        <p class="description">
+         This program's sole purpose has always been to create a safe space for developers
+         worldwide. <br>
+         In the ten years of our existence, we've grown as a community through peer support.
+         This year, we hope to help everyone grow as individuals.
+        </p>
+        <p class="date">2022.04.15(FRI) ~ 16(SAT)</p>
+        <p class="location">@ Bells university of Technology</p>
+      </div>
+    </header>
+    <section class="main-program">
+      <h3 class="section-title">Main Program</h3>
+      <hr class="section-line" />
+      <div class="program-content-nav">
+        <div class="program-content">
+          <img src="images/icons/ignite.png" alt="">
+          <p class="content-title">Ignite</p>
+          <p class="content-description">
+            Get familiar with the latest and best coding practices around.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/lecture.png" alt="" />
+          <p class="content-title">Lecture</p>
+          <p class="content-description">
+            Chance to listen to the top developers from all over the world.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/workshop.png" alt="" />
+          <p class="content-title">Workshop</p>
+          <p class="content-description">
+            Contribute via pair programming with programmers from all walks of life.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/exhibition.png" alt="" />
+          <p class="content-title">Contest</p>
+          <p class="content-description">
+            Go head to head against the top and best developers and fancy your chances of winning.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/forum.png" alt="">
+          <p class="content-title">Discuss</p>
+          <p class="content-description">
+            Questions and Answers session.
+          </p>
+        </div>
+      </div>
+      <a class="join-us" href=""> Join TTPS 2022</a>
+      <a href="index.html" class="see-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="speakers">
+      <h3 class="section-title">Featured Speakers</h3>
+      <hr class="section-line" />
+      <div class="speakers-nav">
+
+      </div>
+      <div class="more-button">
+        <p>MORE</p>
+        <img src="images/arrow-down.svg" alt="down-arrow" />
+      </div>
+    </section>
+    <footer>
+
+    </footer>
+  </main>
+  <script src="top-bar.js"></script>
+  <script src="nav-menu.js"></script>
+  <script src="footer.js"></script>
+  <script src="speakers.js"></script>
```

Comment:
As I mentioned in the previous comment, you should update all the paths:
```html
 <script src="./Javascript Libraries/top-bar.js"></script>
  <script src="./Javascript Libraries/nav-menu.js"></script>
  <script src="./Javascript Libraries/footer.js"></script>
  <script src="./Javascript Libraries/speakers.js"></script>
```
I recommend testing your project on the browser before requesting a new review to ensure all pages, links, and images work as expected.